### ISSUE TITLE
[BUG] bcrypt library failure

### ISSUE THREAD

**Issue by aliavni** [codeforboston/police-data-trust#373]
**Describe the bug**
This happens when building the app in Docker. `bcrypt` library fails with the following error:

```
api-1  | Database 'police_data' already exists.
api-1  | Initialized the database 'police_data'.
api-1  | (trapped) error reading bcrypt version
api-1  | Traceback (most recent call last):
api-1  |   File "/usr/local/lib/python3.11/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
api-1  |     version = _bcrypt.__about__.__version__
api-1  |               ^^^^^^^^^^^^^^^^^
api-1  | AttributeError: module 'bcrypt' has no attribute '__about__'
```

**To Reproduce**
1. Build the app in docker. Since the package version is not pinned, it installs a recent bcrypt version

**Expected behavior**
Docker should build without issues and api container should work.

**Comment by aliavni** [comment]
Here is the PR to fix this issue: https://github.com/codeforboston/police-data-trust/pull/374

**Comment by mikeyavorsky** [comment]
closed with #374
